src/interfaces/store-interfaces/state-interfaces/restoration-state-interface.ts

```diff
@@ -2,22 +2,18 @@ import { ApprovalTypes, RestorationTypes, RelationshipTypes } from '@/enums'
 import { CourtOrderIF } from '@/interfaces/alteration-interfaces'
 
 export interface RestorationStateIF {
+  applicationDate?: string // YYYY-MM-DD
   approvalType: ApprovalTypes
-  approvalTypeValid: boolean
-  businessNameValid: boolean
```

Comment:
These are totally "decoys" and wasted a not-insignificant amount of my time. Instead of these, the `companyFlags` object is used.